libs/Microsoft.MixedReality.WebRTC.Native/src/api.cpp

```diff
@@ -1075,6 +1075,34 @@ void MRS_CALL mrsPeerConnectionRemoveLocalAudioTrack(
   }
 }
 
+
+
+void MRS_CALL mrsAudioReadStreamCreate(PeerConnectionHandle peerHandle,
```

Comment:
API functions like this generally return an `mrsResult` so that we can transform errors into exceptions in C# for example.